src/jdk.compiler/share/classes/com/sun/tools/javac/resources/compiler.properties

```diff
@@ -1764,6 +1764,9 @@ compiler.warn.dir.path.element.not.directory=\
 compiler.warn.missing-explicit-ctor=\
     class {0} in exported package {1} declares no explicit constructors, thereby exposing a default constructor to clients of module {2}
 
+compiler.warn.strictfp=\
+    as of release 17, all floating-point expressions are evaluated strictly and ''strictfp'' is not required
+
```

Comment:
Nitpick: Three other uses of floating point in the same file use the non-hyphenated form.

Reply:
As an adjective "floating-point" rather than "floating point" is the more correct construction. JLS uses "floating-point" consistently. I'd prefer to change the other three occurrences to "floating-point" for consistency rather than changing this one to "floating point".

Reply:
Pushed an update to use "floating-point" consistently.